src/os/exec.go

```diff
@@ -47,6 +47,10 @@ func (p *ProcessState) Sys() interface{} {
 	return nil // TODO
 }
 
+func (p *ProcessState) Exited() bool {
+	return false // TODO
+}
```

Comment:
Please don't implement functions like this. This could break existing programs that rely on `Exited` returning a correct value.

(This is different from `Sys()` for example, where programs would type assert anyway and the type assert would probably fail - therefore clearly indicating the issue).

Reply:
So what do you propose to do here? The ProcessState afaik is just a stub, so the functions implemented over this struct should also be stubs. Re-implementing the process model of tinygo would definitely overshoot this PRs scope.